test/routes/LandingPage.test.jsx

```diff
@@ -0,0 +1,24 @@
+import { render, cleanup } from '@testing-library/react';
+import React from 'react';
+import { expect, it, afterEach, describe } from 'vitest';
+import { ThemeProvider } from '@mui/material/styles';
+
+import LandingPage from '../../src/routes/LandingPage';
+import theme from '../../src/theme';
+
+const LandingPageWithContexts = () => (
+  <ThemeProvider theme={theme}>
+    <LandingPage />
+  </ThemeProvider>
+);
+
+describe('Landing Page', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('renders', () => {
+    const { container } = render(<LandingPageWithContexts />);
+    expect(container).toMatchSnapshot();
```

Comment:
Most of the size of this PR comes from this one snapshot, and that snapshot is really hard to read.  I'm still not sure of the best way to handle snapshotting with react test library. Generally I think we should only snapshot components that generate HTML directly, since that's what the user will see. We shouldn't really need to snapshot components that simply render other react components. However, since we're using MUI, very little of our codebase generates HTML, so that differentiation may break down.

Reply:
Refactored this so the snapshot isn't as large. This Snapshot works better now.